tests/phpunit/tests/admin/wpUpgrader.php

```diff
@@ -0,0 +1,1430 @@
+<?php
+/**
+ * Tests the `WP_Upgrader` class.
+ *
+ * @package WordPress
+ *
+ * @since 6.1.0
+ */
+
+/**
+ * Tests_Admin_WpUpgrader class.
+ *
+ * @group admin
+ * @group upgrade
+ *
+ * @covers WP_Upgrader
```

Comment:
```suggestion
 * @coversDefaultClass \WP_Upgrader
```

The methods have individual `@covers` tags, so having a class level `@covers` doesn't make sense - which one is valid ?

Using `@coversDefaultClass` will allow to shorten the method level `@covers` tags though to just the method like so `@covers ::method_name`.

Also note the leading backslash: `@covers` tags **must** contain fully qualified names, so it is good practice to make this super-clear.

Reply:
Didn't know that `@coversDefaultClass` allows shortening of the method level `@covers`! 🤯

Reply:
For reference: https://phpunit.readthedocs.io/en/9.5/annotations.html#coversdefaultclass